html/controls/plt/plt-input.js

```diff
@@ -338,9 +341,16 @@ function preparePltInputTable(element, teamId, mode, statsbookPeriod, alternateN
 			p.append(boxCell);
 			
 			var advanceCell = $('<td>').addClass('Advance').attr('rowspan', 2).click(function() {
-				WS.Set('ScoreBoard.Team('+t+').AdvanceFieldings', true);
+				if (isTrue(WS.state['ScoreBoard.Team('+t+').FieldingAdvancePending'])) {
+					WS.Set('ScoreBoard.Team('+t+').AdvanceFieldings', true);
+				} else if (advanceCell.hasClass('OnTrack')) {
+					openAnnotationEditor(t, id);
```

Comment:
You can't edit annotations from a jam that has just ended from here, as the advance is showing.

There's also no indication on the LT sheet that there's an annotation.

Reply:
Advance overriding the annotation field was a tradeoff I consciously chose:
- The cases where you need to enter an annotation just as the jam ends are exceedingly rare. And if one arises, you can still enter the annotation in the fielding editor.
- If a user were to enter a substitution before pressing "Advance", the substitution would be entered for the previous interjam. This user error is prevented here.
- There is no other reasonable spot to put either element.

Reply:
Okay. Given you're probably on a tablet, typing in "Penalty Overturned" would be tricky. We should probably have buttons for those on the fielding editor too.